docs/DeveloperGuide.adoc

```diff
@@ -297,7 +297,8 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 (For all use cases below, the *System* is `Cooking Papa` and the *Actor* is the `user`, unless specified otherwise)
 
 [source,shell]
-----
+
+```
 Use case: Create a recipe
```

Comment:
Should we include a unique index for each UC (e.g. UC01) similar to the ones shown in the textbook?

Reply:
i'll make mine UC01